sync/optimistic.md

```diff
@@ -270,6 +270,23 @@ optimistic blocks (and vice-versa).
 
 ## Design Decision Rationale
 
+### Why sync optimistically?
+
+Most execution engines use state sync as a default sync mechanism on Ethereum Mainnet 
+because executing blocks from genesis takes several weeks on commodity hardware.
+
+State sync requires the knowledge of the current head of the chain to converge eventually.
+If not constantly fed with the most recent head, state sync won't be able to complete
+because the recent state soon becomes unavailable due to state trie pruning.
+
+Optimistic block import (i.e. import without waiting for a payload to be executed by the engine)
+breaks a deadlock between the state sync and processing a beacon block by skipping payload
+execution as long as execution engine is syncing.
+
+Optimistic sync works nicely with execution engines using block execution as a default
```

Comment:
It not only works nicely but is also requisite (unless CL and EL lockstep execute from genesis

Reply:
Block sync allows for a different initial sync strategy. CL may send a checkpoint payload to EL and wait until it catches up with this payload then lock-step - but this would be suboptimal. I agree that "nicely" might be too weak word in this context, but optimistic sync is not a requisite for block sync either. Will try to rephrase this statement

Reply:
Done

Reply:
@djrtwo waiting for you to take a look at the update of this statement and pushing the button if OK with you